search-app/resources/templates/eosdis-landing-links.html

```diff
@@ -0,0 +1,9 @@
+{% extends "templates/base.html" %}
+
+{% block title %}CMR Search - Landing Pages for EOSDIS Collections{% endblock %}
```

Comment:
Changing ambiguous page header above links from "Landing Pages for EOSDIS Collections" to "Directory of Landing Pages for EOSDIS Collections", etc.